spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -367,10 +389,10 @@ public static void configureTls(ServerBuilder sb, Ssl ssl,
                     .forServer(getKeyManagerFactory(ssl, keyStoreSupplier))
                     .trustManager(getTrustManagerFactory(ssl, trustStoreSupplier));
             final SslProvider sslProvider = ssl.getProvider() == null ?
-                    Flags.useOpenSsl() ?
-                            SslProvider.OPENSSL
-                            : SslProvider.JDK
-                    : ssl.getProvider();
+                                            Flags.useOpenSsl() ?
+                                            SslProvider.OPENSSL
+                                                               : SslProvider.JDK
+                                                                      : ssl.getProvider();
```

Comment:
How about using `if`? it's hard for me to recognize when there are two ternaries.
```java
final SslProvider sslProvider;
final SslProvider provider = ssl.getProvider();
if (provider != null) {
    sslProvider = provider;
} else {
    sslProvider = Flags.useOpenSsl() ? SslProvider.OPENSSL : SslProvider.JDK;
}
```